Dockerfile

```diff
@@ -3,7 +3,7 @@ WORKDIR /go/src/github.com/openshift/cluster-api-provider-gcp
 COPY . .
 # VERSION env gets set in the openshift/release image and refers to the golang version, which interfers with our own
 RUN unset VERSION \
- && NO_DOCKER=1 make build
+ && make build NO_DOCKER=1 CGO_ENABLED=0
```

Comment:
Makefile:11-19
```
NO_DOCKER ?= 0
ifeq ($(NO_DOCKER), 1)
  DOCKER_CMD =
  IMAGE_BUILD_CMD = imagebuilder
  CGO_ENABLED = 1
else
  DOCKER_CMD := docker run --rm -e CGO_ENABLED=1 -v "$(PWD)":/go/src/github.com/openshift/cluster-api-provider-gcp:Z -w /go/src/openshift/cluster-api-provider-gcp openshift/origin-release:golang-1.12
  IMAGE_BUILD_CMD = docker build
endif
```

I don't think we use any C libraries directly in the code. So we might just set CGO_ENABLED to 0 by default.

Reply:
Which other archs are we trying to support?

Reply:
> I don't think we use any C libraries directly in the code. So we might just set CGO_ENABLED to 0 by default.

The `CGO_ENABLED = ` is unconditional. Change it to `CGO_ENABLED ?= 1`.

Reply:
> Which other archs are we trying to support?

IBM Power LE, for [IBM Power Systems for Google Cloud](https://console.cloud.google.com/marketplace/details/ibm-sg/ibm-power-cloud-for-gcp).

Reply:
> The `CGO_ENABLED = ` is unconditional. Change it to `CGO_ENABLED ?= 1`.

The difference in syntax only changes how the variable can be overridden.  `?=` allows also for environment variable overrides (e.g. `NO_DOCKER=1 make`), `=` allows only for argument overrides (e.g. `make CGO_ENABLED=0`).